### PR TITLE
docs(forms): code shuffle, text, & alpha52 updates

### PR DESCRIPTION
**Pull request by wardbell**

### CODE REVIEW

File: public/docs/ts/latest/guide/forms.jade [hunk 1]
```diff
@@ -51,7 +50,7 @@ include ../../../../_includes/_util-fns
   We'll discuss and learn to build the following template-driven form:
 
 figure.image-display
-  img(src="/resources/images/devguide/forms/hf-1.png" width="400px" alt="Clean Form")
+  img(src="/resources/images/devguide/forms/heroForm-1.png" width="400px" alt="Clean Form")
```
Comment (kwalrath) [inline public/docs/ts/latest/guide/forms.jade]: heroForm -> hf

File: public/docs/ts/latest/guide/forms.jade [hunk 2]
```diff
@@ -62,7 +61,7 @@ figure.image-display
   If we delete the hero name, the form displays a validation error in an attention grabbing style:
 
 figure.image-display
-  img(src="/resources/images/devguide/forms/hf-2.png" width="400px" alt="Invalid, Name Required")
+  img(src="/resources/images/devguide/forms/heroForm-2.png" width="400px" alt="Invalid, Name Required")
```
Comment (kwalrath) [inline public/docs/ts/latest/guide/forms.jade]: heroForm -> hf

File: public/docs/ts/latest/guide/forms.jade [hunk 3]
```diff
@@ -239,11 +239,11 @@ ol
 
 .l-main-section
 :marked
-  ## Two-way data binding with ***ng-model**
+  ## Two-way data binding with ***ngModel**
   Running the app right now would be disappointing.
 
 figure.image-display
-  img(src="/resources/images/devguide/forms/hf-3.png" width="400px" alt="Early form with no binding")
+  img(src="/resources/images/devguide/forms/heroForm-3.png" width="400px" alt="Early form with no binding")
```
Comment (kwalrath) [inline public/docs/ts/latest/guide/forms.jade]: heroForm -> hf

File: public/docs/ts/latest/guide/forms.jade (hunk 1 above)
Comment (wardbell) [inline public/docs/ts/latest/guide/forms.jade]: I won't change the image names. Too much trouble for too little gain.

**Pull request closed without merging**